### PR TITLE
Docs for web 'Open with' configuration

### PR DESCRIPTION
**Pull request by will-moore**
Documentation for using new "Open with" options.
PR open but not yet merged at https://github.com/openmicroscopy/openmicroscopy/pull/4630

To test, configure 'Open with' using all the different options described.

### CODE REVIEW

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (will-moore) [inline omero/developers/Web/LinkingFromWebclient.txt:161]: Travis is currently failing with 
[exec] /home/travis/build/openmicroscopy/ome-documentation/omero/developers/Web/LinkingFromWebclient.txt:158: ERROR: Unexpected indentation.
@hflynn any ideas what I'm doing wrong here?

**Comment by will-moore** [comment]
@waxenegger I wonder if you could add any comments you have on the docs. Clear enough?
Thanks.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (waxenegger) [reply omero/developers/Web/LinkingFromWebclient.txt:161]: @will-moore sorry missed that bit. couldn't complain about docs.

I used them when testing the open with functionality, adding scripts and enabled easy enough, as well as url and options such as target to set everything up before I started looking into the code for more analysis. I missed your mix-up in the code 'supported_objects' (which you fixed meanwhile) but that does not concern the documentation. You covered supported objects there.

So this is good to go I'd say.

Comment (hflynn) [reply omero/developers/Web/LinkingFromWebclient.txt:161]: Not sure what Travis is complaining about, I'll fetch this branch and see if I can work it out

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt:149]: @will-moore you need to add a blank line after the `::` before your lines of text then it should build okay

**Review by hflynn — CHANGES_REQUESTED** [review]
Formatting fix needed to get Travis green

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (will-moore) [reply omero/developers/Web/LinkingFromWebclient.txt:149]: Done, thanks.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: To match the case of the other titles, this should be 'Linking from Webclient' with a lowercase 'f'

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: These bullet points shouldn't be indented

File: omero/developers/Web/CreateApp.txt
Comment (hflynn) [inline omero/developers/Web/CreateApp.txt]: Subheadings should only have an initial capital to match the rest of the page - 'Linking from webclient'

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: E.g. -> e.g.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: E.g. -> e.g.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: E.g. -> e.g.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: I'm not a fan of E.g. to start a sentence, replace with 'For example' please.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: As this is a single line example, it would be neater to put the `::` after `images` so you don't need all these extra lines, just one blank line and then the example.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: E.g. -> e.g.

**Review by hflynn — COMMENTED** [review]
Sorry, a few more nitpicks from having looked over the page now it's built on staging. I guess the 'E.g.'s can stay in your codeblock examples since they start sentences and writing out 'For example' here seems excessive, but in the text, I'd prefer it written out properly when starting sentences and it should be lowercase when within a sentence.

**Comment by will-moore** [comment]
@hflynn I updated text to reflect recent change in https://github.com/openmicroscopy/openmicroscopy/pull/4630 and also addressed your comments.

**Review by hflynn — APPROVED** [review]

**Comment by hflynn** [comment]
Formatting looks good now thanks. I'll hold off merging until the related code PR is merged just incase there are further tweaks.

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: this will need updating to 'omero-webtest' now

Comment (hflynn) [reply omero/developers/Web/LinkingFromWebclient.txt]: Although this is redirecting anyway. But still good to update it.

**Comment by pwalczysko** [comment]
Where is this built please ? http://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web/CreateApp.html ???

**Comment by hflynn** [comment]
http://www.openmicroscopy.org/site/support/omero5.3-staging/developers/Web/LinkingFromWebclient.html mostly @pwalczysko

**Comment by hflynn** [comment]
Closing temporarily to fix the broken webtest links, will re-open once the branch is merged into this and the conflict fixed.

**Comment by hflynn** [comment]
Actually, scratch that, it's the 5.2 branch that is broken, sorry for the confusion!

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [reply omero/developers/Web/LinkingFromWebclient.txt]: Actually, as per the dev_5_2 branch, the link should be `OMERO.webtest` with the updated omero-webtest URL

**Review by hflynn — CHANGES_REQUESTED** [review]
webtest URL still needs updating

**Comment by will-moore** [comment]
Comment from @waxenegger "I believe the documentation is good, in general. I like the code snippets, they helped me most.
I did forget about syncing after I created a script file in the plugin which is something that I do naturally when I develop. Perhaps it's that this was more of a configuration task almost that I forgot it was necessary. People who might approach it from that direction could forget as well and think all that was needed is call set/append config although one could argue that ideally the files for open with are/should be already part of the plugin.
Don't know of if a mention of restarting/syncing web is useful or not...up to you."

Harald - I've pushed a commit for this if you want to have a look, Thanks.

**Comment by jburel** [comment]
Code PR has been merged

File: omero/developers/Web/LinkingFromWebclient.txt
Comment (hflynn) [inline omero/developers/Web/LinkingFromWebclient.txt]: You've got an extra set of backticks here

**Comment by hflynn** [comment]
One further minor formatting issue from me. Does this need further technical review tomorrow?

**Comment by will-moore** [comment]
@hflynn Thanks, fixed backticks. I'll list for final review tomorrow.

**Review by hflynn — APPROVED** [review]

**Comment by waxenegger** [comment]
all good, thanks for adding the restart/sync remark.

**Pull request merged**
